flocker/acceptance/testtools.py

```diff
@@ -265,15 +267,19 @@ def get_nodes(test_case, num_nodes):
     # Only return the desired number of nodes
     reachable_nodes = set(sorted(reachable_nodes)[:num_nodes])
 
-    # Remove all existing containers; we make sure to pass in node
-    # hostnames since we still rely on flocker-deploy to distribute SSH
-    # keys for now.
-    clean_deploy = {u"version": 1,
-                    u"nodes": {node: [] for node in reachable_nodes}}
     clean_applications = {u"version": 1,
                           u"applications": {}}
-    flocker_deploy(test_case, clean_deploy, clean_applications)
-    getting = get_test_cluster()
```

Comment:
Oh.  Sorry, ignore the second part of that.  This is another helper, not a test method.  I misread.